tests/framework/installer/ceph_installer.go

```diff
@@ -352,6 +352,11 @@ func (h *CephInstaller) UninstallRookFromMultipleNS(helmInstalled bool, systemNa
 	logger.Infof("Uninstalling Rook")
 	var err error
 	for _, namespace := range namespaces {
+		if h.T().Failed() {
```

Comment:
@travisn @galexrt what do you think about printing the pod descriptions based on whether the test has failed? I see the `if ... <CephInstaller>.T().Failed()` pattern used a few times elsewhere, so this seems like it might be good here too. And I think teardown of namespaces seems like a good, easy place to have this code since the `PrintPodDescribeForNamespace` method already exists, as does the loop over namespaces.

Reply:
yes, that should work well here, much better than always printing it.